universal/pm.md

```diff
@@ -43,7 +43,13 @@ XCPM does not natively support Haswell-E and Broadwell-E, this means we need to
     * Cpuid1Data: `D4060300 00000000 00000000 00000000`
     * Cpuid1Mask: `FFFFFFFF 00000000 00000000 00000000`
 
-## Using CPU Friend
+## Manually Modifying Frequency vectors
+
+In most cases, the native CPU power management data shipped with macOS works out of the box. If you're experiencing issues, changing SMBIOS to something more appropriate to your system will provide different frequency vectors and may be better for your usecase. In the cases where manual tuning is required (unsupported CPU or a CPU not used in any Macs), you can use CPUFriend to inject modified frequency vectors, but if you don't know what you're doing, you can severely break power management.
```

Comment:
> a CPU not used in any Macs

There are only very limited numbers of CPUs used by real Macs. In my opinion, just drop the parentheses; skilled users know what they are doing.